src/js/AlertButtons.js

```diff
@@ -29,7 +29,19 @@ export default class AlertButtons extends React.Component {
         var softButtons = this.props.alertButtons
         var items;
         if(!softButtons){
-            return (null)
+            let okButton = {
```

Comment:
We should add `msgID` property so that a response will be sent by `uiController.onDefaultAction`: https://github.com/smartdevicelink/generic_hmi/blob/76aea6ea482e0b18e7cf59b6cb54509e0c880783/src/js/Controllers/UIController.js#L852